tutorial/source/intro_part_i.ipynb

```diff
@@ -43,58 +43,21 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Variable containing:\n",
-      " 1.6869\n",
-      "[torch.FloatTensor of size 1]\n",
-      "\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "mu = Variable(torch.zeros(1))   # mean zero\n",
-    "sigma = Variable(torch.ones(1)) # unit variance \n",
-    "x = dist.normal(mu, sigma)      # x is a sample from N(0,1)\n",
-    "print(x)"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "Importantly, `dist.normal` is a _function_ that takes parameters and `dist.normal(...)` returns a sample. Note that the parameters passed to `dist.normal` are PyTorch `Variables`. This is necessary because we want to make use of PyTorch's fast tensor math and autograd capabilities. To score the sample `x`, i.e. to compute its log probability according to the distribution $\\mathcal{N}(0,1)$, we simply do the following:"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Variable containing:\n",
-      "-2.3417\n",
-      "[torch.FloatTensor of size 1]\n",
-      "\n"
-     ]
-    }
-   ],
-   "source": [
-    "log_p_x = dist.normal.log_pdf(x, mu, sigma)\n",
-    "print(log_p_x)"
+    "sigma = Variable(torch.ones(1)) # unit variance\n",
+    "normal = dist.Normal(mu, sigma) # create a normal distribution object\n",
+    "x = normal.sample() # draw a sample from N(0,1)\n",
+    "print(\"sample\", x)\n",
+    "print(\"log prob\", normal.log_pdf(x)) # score the sample from N(0,1)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Note that the arguments to the `log_pdf` method are: first, the value to score (i.e. `x`), followed by the arguments to the distribution (i.e. `mu` and `sigma`)."
+    "Here, `dist.Normal` is a callable object that takes parameters and provides sample and score methods. Note that the parameters passed to `dist.Normal` are PyTorch `Variables`. This is necessary because we want to make use of PyTorch's fast tensor math and autograd capabilities during inference."
```

Comment:
i think we probably need larger changes here... since the whole stochastic function story is now a bit different

Reply:
I don't think it's different, things still work the same way under the hood.

Reply:
well, don't we need to be more careful. something like

Here we construct an instance `normal` of `dist.Normal`, which is a `Distribution`. `normal` supports `sample` and `log_pdf` methods and is itself directly callable, since its `__call__` method points to `sample`

or whatever

also this needn't be true anymore:

Note that the parameters passed to dist.Normal are PyTorch Variables

Reply:
i reworded it but i dont think its different. distributions are still stochastic functions that have methods that can be called.

>this needn't be true anymore

its still needed to do inference and learn